inst/examples/spinner-ex.R

```diff
@@ -0,0 +1,27 @@
+library(scales) # for the alpha() function
```

Comment:
It is not worth introducing this new dependency only for an example (and only one function in **scales** is used).